src/mixins/meta.ts

```diff
@@ -127,3 +155,51 @@ export function metaTag(
 
   return metaTagsObject;
 }
+
+// ALernate locales generator
+export interface AlterateLocalLink {
+  href: string;
+  hreflang: string;
+  rel: "alternate";
+}
+
+export interface OpenGraphMetaTag {
+  content: string;
+  property: string;
+}
+
+function pageAlternateLocales(locales: QMetaI18nAlternateLocale[], useOpengraph?: QMetaI18nAlternateLocale) {
+  const currentDomain = domain();
```

Comment:
Non so quanto abbia senso metterlo in una variabile locale, lo usi una sola volta

Reply:
Well, actually no. The domain is used inside a _for cycle_, so storing the return value of _domain()_ inside a variable will save some function calls.